cmd/glasskube/cmd/install.go

```diff
@@ -118,15 +119,48 @@ var installCmd = &cobra.Command{
 
 		pkgBuilder.WithVersion(installCmdOptions.Version)
 
-		installationPlan := []dependency.Requirement{
-			{PackageWithVersion: dependency.PackageWithVersion{Name: packageName, Version: installCmdOptions.Version}},
-		}
-
 		var manifest v1alpha1.PackageManifest
 		if err := repoClient.FetchPackageManifest(packageName, installCmdOptions.Version, &manifest); err != nil {
 			fmt.Fprintf(os.Stderr, "❗ Error: Could not fetch package manifest: %v\n", err)
 			cliutils.ExitWithError()
-		} else if validationResult, err :=
+		}
+
+		installationPlan := []dependency.Requirement{}
+		if manifest.Scope.IsCluster() {
+			if len(args) != 1 {
+				fmt.Fprintf(os.Stderr,
+					"❌ %v has scope Cluster. Specifying an instance name for a ClusterPackage is not possible\n",
+					packageName)
+				cliutils.ExitWithError()
+			}
+			installationPlan = append(installationPlan,
+				dependency.Requirement{PackageWithVersion: dependency.PackageWithVersion{
+					Name:    packageName,
+					Version: installCmdOptions.Version,
+				}},
+			)
+		} else {
+			var name string
+			if len(args) != 2 {
+				fmt.Fprintf(os.Stderr, "%v has scope Namespaced. Please enter a name (default %v):\n", packageName, packageName)
+				name = cliutils.GetInputStr("name")
+				if name == "" {
+					name = packageName
+				}
+			} else {
+				name = args[1]
+			}
+			ns := installCmdOptions.GetActualNamespace(ctx)
```

Comment:
Installing with the `--namespace` argument does not use the given namespace, because `.GetActualNamespace` is accessing `configureCmdOptions`.